src/services/completions.ts

```diff
@@ -1105,75 +1100,6 @@ function getExhaustiveCaseSnippets(
     return undefined;
 }
 
-interface CaseClauseTracker {
-    addValue(value: string | number): void;
-    hasValue(value: string | number | PseudoBigInt): boolean;
-}
-
-function newCaseClauseTracker(checker: TypeChecker, clauses: readonly (CaseClause | DefaultClause)[]): CaseClauseTracker {

```

Comment:
I think it makes sense to move this out to a shared space so both `completions` and `stringCompletions` can use it - but if you prefer some other solution, let me know.